build.js

```diff
@@ -260,19 +260,19 @@ async function copyStatic() {
       { overwrite: false, recursive: true }
     ),
 
-    fsExtra.copyFile(
+    fsExtra.copy(
       path.join(
         __dirname,
         'node_modules/jquery.fancytable/dist/fancyTable.min.js'
       ),
       path.join(__dirname, 'build/static/js/fancyTable.min.js'),
-      fs.constants.COPYFILE_EXCL | fs.constants.COPYFILE_FICLONE
+      { overwrite: false }
```

Comment:
The file is originally fixed and it doesn't seem to change it because it's a foundation one....Hmmmm. It seems better to put them into node_modules and refer them directly from codes. Thanks I'll change it.

Reply:
Ah thanks and fixed it.